packages/openapi-typescript/test/node-api.test.ts

```diff
@@ -465,6 +465,34 @@ export type operations = Record<string, never>;`,
                 type: "number",
                 enum: [100, 101, 102, 103, 104, 105],
               },
+              XEnumVarnames: {
```

Comment:
This is not a great test, as it’s hidden inside another test.

For this, we should add a net-new test so that we’re testing explicitly this usecase. The test you added this to is testing the `enum` field, NOT `x-enumNames` and `x-enumDescriptions`. If there were any regression in the `x-enumNames` code, we’d want to see a test fail that flagged that specifically.